x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/findings_by_resource_container.tsx

```diff
@@ -46,7 +54,8 @@ export const FindingsByResourceContainer = ({ dataView }: { dataView: DataView }
 const LatestFindingsByResource = ({ dataView }: { dataView: DataView }) => {
   useCspBreadcrumbs([findingsNavigation.findings_by_resource]);
 
-  const { urlQuery, setUrlQuery } = useUrlQuery(getDefaultQuery);
+  const getPersistedDefaultQuery = usePersistedQuery(getDefaultQuery);
+  const { urlQuery, setUrlQuery } = useUrlQuery(getPersistedDefaultQuery);
```

Comment:
`useUrlQuery` expects a memozied callback, and we need to inject the current app filters to `getDefaultQuery` so we do that in `usePersistedHook` which injects + wraps in `useCallback`